### DISCUSSION TITLE
Card: ABM Treaty withdrawal

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#7] [Ideas]
- Clarification: Original text had "(or 2 the East Block)". Does this refer to a 2p game where one player represent the whole East Block?
- Yes
- Then similar to Neocolonialism?
    > US loses 3 VP.
    > **Only one East player partaking**: East player loses 2 VP.
    > **Otherwise**: Russia and China lose 1 VP each.

**Comment by RebelusQuo** [comment]
Confirmed?

**Reply by rodrigoSantamaria** [reply]
Yes, confirmed, i missed this one, sorry.
